tests/mypy/modules/success.py

```diff
@@ -113,3 +103,17 @@ class WrapperModel(GenericModel, Generic[T]):
 class WithField(BaseModel):
     age: int
     first_name: str = Field('John', const=True)
+
+
+class Config:
+    validate_assignment = True
+
+
+@dataclass(config=Config)
+class AddProject:
```

Comment:
this should be removed since it fails and `tests/mypy/outputs/success.txt` should not exist.

Reply:
I removed it.

Reply:
@samuelcolvin @koxudaxi When I named this file originally, the idea was that `success.py` would run without exceptions, and other files wouldn't necessarily run without exceptions. It didn't mean that mypy wouldn't necessarily complain about things inside it, it was just a signal that we could check that mypy wasn't reporting success when the file wouldn't actually run.

But I'm amenable to other conventions here (especially since this one clearly wasn't obvious).

Reply:
@dmontagu 
I'm sorry, I should have discussed it with quoted your comment before I did it.